apps/ledger-live-mobile/src/screens/SyncOnboarding/ResyncOverlay.tsx

```diff
@@ -47,12 +48,47 @@ const ResyncOverlay = ({ isOpen, delay = 0, productName }: Props) => {
     <Flex
       zIndex={100}
       position="absolute"
-      backgroundColor="constant.overlay"
       top={0}
       left={0}
       height="100%"
       width="100%"
     >
+      <Svg width="100%" height="100%">
+        <Defs>
+          <LinearGradient
+            id="resyncOverlayGradient"
+            x1="0%"
+            y1="0%"
+            x2="0%"
+            y2="100%"
+            gradientUnits="userSpaceOnUse"
+          >
+            <Stop offset="0%" stopOpacity={0} stopColor="transparent" />
+            <Stop
+              offset="10%"
+              stopOpacity={0.4}
+              stopColor={colors.constant.overlay}
+            />
+            <Stop
+              offset="50%"
+              stopOpacity={0.5}
+              stopColor={colors.constant.overlay}
```

Comment:
same

Reply:
can you use `"constant.overlay"` ? or get `colors` form `useTheme` if it does not work